praw/models/reddit/subreddit.py

```diff
@@ -628,9 +620,7 @@ def _submit_media(self, data, timeout, without_websockets):
         ) as ws_exception:
             raise WebSocketException(
                 "Websocket error. Check your media file. "
-                "Your post may still have been created. "
-                "Use this exception's .original_exception attribute to get "
-                "the original exception.",
+                "Your post may still have been created.",
```

Comment:
Is `post` still accurate here? Or can this be used to also submit media elsewhere?

Reply:
It's still accurate. The `_submit_media` method is only used for creating video submissions and photo submissions.